<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VMT10288" ignored="0" name="Test" tests="2" failures="2" timestamp="2021-03-19T07:45:42 IST" time="0.161" errors="0">
  <testcase name="amazon" time="0.014" classname="com.valuemomentum.retail.TestAutomation.AssessmentDemo">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	Unreachable catch block for IOException. This exception is never thrown from the try statement body
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
Unreachable catch block for IOException. This exception is never thrown from the try statement body

at com.valuemomentum.retail.TestAutomation.AssessmentDemo.amazon(AssessmentDemo.java:172)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- amazon -->
  <testcase name="medpluss" time="0.002" classname="com.valuemomentum.retail.TestAutomation.AssessmentDemo">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	Unreachable catch block for IOException. This exception is never thrown from the try statement body
	Unreachable catch block for IOException. This exception is never thrown from the try statement body
	Unreachable catch block for IOException. This exception is never thrown from the try statement body
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
Unreachable catch block for IOException. This exception is never thrown from the try statement body
Unreachable catch block for IOException. This exception is never thrown from the try statement body
Unreachable catch block for IOException. This exception is never thrown from the try statement body

at com.valuemomentum.retail.TestAutomation.AssessmentDemo.medpluss(AssessmentDemo.java:75)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- medpluss -->
</testsuite> <!-- Test -->
